.devops/azure-templates/make-build-steps.yml

```
@@ -0,0 +1,57 @@
+# Azure DevOps pipeline template used to checkout, install node dependencies and build the code. 
+
+parameters:
+- name: 'make'
+  type: string
+  default: install_dependencies
+  values:
+  - install_dependencies
+  - build
+  - predeploy_build
+
+- name: 'cache_version_id'
+  type: string
+  default: $(CACHE_VERSION_ID)
+
+# the branch, tag or commit to deploy
+- name: 'gitReference'
+  type: string
+  default: '$(Build.SourceVersion)'
+  
+steps:
+- checkout: self
+  displayName: 'Checkout'
+
+- task: Cache@2
+  inputs:
+    key: 'yarn-${{ parameters.cache_version_id }} | "$(Agent.OS)" | yarn.lock'
+    restoreKeys: |
+      yarn-${{ parameters.cache_version_id }} | "$(Agent.OS)"
+    path: $(YARN_CACHE_FOLDER)
+  displayName: 'Cache yarn packages'
+
```

add this 
```suggestion
cacheHitVar: CACHE_RESTORED
```
here why https://github.com/pagopa/azure-pipeline-templates/pull/2#issuecomment-734829613